### PR TITLE
Fixing skybox bug

### PR DESCRIPTION
**Pull request by andreibosco**
Creating a custom GradientSkybox shader that supports Single Pass Instanced rendering mode

### CODE REVIEW

**Comment by provencher** [comment]
Thanks for this PR @andreibosco!

File: Assets/MixedRealityToolkit.ThirdParty/OculusQuestInput/Art/Shaders/GradientSkybox.shader [hunk 1]
```diff
@@ -0,0 +1,86 @@
...
+Copyright (c) Facebook Technologies, LLC and its affiliates. All rights reserved.  
+
+See SampleFramework license.txt for license terms.  Unless required by applicable law 
+or agreed to in writing, the sample code is provided “AS IS” WITHOUT WARRANTIES OR 
+CONDITIONS OF ANY KIND, either express or implied.  See the license for specific 
+language governing permissions and limitations under the license.
+
+************************************************************************************/
+
+Shader "Custom/Procedural Gradient Skybox"
```
Comment (provencher) [inline Assets/MixedRealityToolkit.ThirdParty/OculusQuestInput/Art/Shaders/GradientSkybox.shader]: Please change the Shader folder to be "MRTK-Quest/Procedural Gradient SkyBox" to differentiate it from the source shader.

File: Assets/MixedRealityToolkit.ThirdParty/OculusQuestInput/Art/Shaders/GradientSkybox.shader [hunk 2]
```diff
@@ -0,0 +1,86 @@
+﻿/************************************************************************************
+
+Copyright (c) Facebook Technologies, LLC and its affiliates. All rights reserved.  
+
+See SampleFramework license.txt for license terms.  Unless required by applicable law 
+or agreed to in writing, the sample code is provided “AS IS” WITHOUT WARRANTIES OR 
+CONDITIONS OF ANY KIND, either express or implied.  See the license for specific 
+language governing permissions and limitations under the license.
```
Comment (provencher) [inline Assets/MixedRealityToolkit.ThirdParty/OculusQuestInput/Art/Shaders/GradientSkybox.shader:8]: If you could add your changes to the disclaimer, header, that would be appreciated.
If you could also point to the directory of the original, that would be great.

**Review by provencher — APPROVED** [review]
Approved pending some minor changes.

Merge it when ready.

File: Assets/MixedRealityToolkit.ThirdParty/OculusQuestInput/Art/Shaders/GradientSkybox.shader (hunk 1 above)
Comment (andreibosco) [reply Assets/MixedRealityToolkit.ThirdParty/OculusQuestInput/Art/Shaders/GradientSkybox.shader]: Done

File: Assets/MixedRealityToolkit.ThirdParty/OculusQuestInput/Art/Shaders/GradientSkybox.shader (hunk 2 above)
Comment (andreibosco) [reply Assets/MixedRealityToolkit.ThirdParty/OculusQuestInput/Art/Shaders/GradientSkybox.shader:8]: I've updated the header with the requested information.

**Pull request merged**